src/main/resources/assets/appliedenergistics2/lang/de_de.lang

```diff
@@ -49,23 +49,23 @@ tile.appliedenergistics2.crafting_storage_16k.name=16k-Fertigungsspeicher
 tile.appliedenergistics2.crafting_storage_64k.name=64k-Fertigungsspeicher
 tile.appliedenergistics2.crafting_accelerator.name=Fertigungs-Prozessoreinheit
 tile.appliedenergistics2.crafting_unit.name=Fertigungseinheit
-tile.appliedenergistics2.molecular_assembler.name=Molekularassembler
+tile.appliedenergistics2.molecular_assembler.name=Molekularmonteur
```

Comment:
assembler is the english word, I dont have to explain much here

Reply:
no .. just no .. "Molekularfabrikator" maybe ..

Reply:
good idea... changed it